src/checker.ts

```diff
@@ -116,7 +64,15 @@ export function attachNodesToAssertions(
 
   collectNodes(source);
   if (assertions.length) {
-    console.error(assertions);
+    let prettyAssertions = assertions.map(o => {
```

Comment:
this can be written more directly and with fewer type casts:

```ts
const prettyAssertions = assertions.map(assertion => {
  let msg: string;
  if (assertion.kind === 'type') {
    msg = `$ExpectType ${assertion.type}`;
  } else {
    msg = `$ExpectError ${assertion.pattern}`
  }
  return `{assertion.line + 1}: ${msg}`;
});
```